seed/landing/views.py

```diff
@@ -124,3 +133,69 @@ def signup(request, uidb64=None, token=None):
         set_password_form=SetPasswordForm,
         post_reset_redirect=reverse('landing:landing_page') + "?setup_complete"
     )
+
+
+def create_account(request):
+    if request.method == "POST":
+        redirect_to = request.POST.get('next', request.GET.get('next', False))
+        if not redirect_to:
+            redirect_to = reverse('seed:home')
+        form = CustomCreateUserForm(request.POST)
+        errors = ErrorList()
+        if form.is_valid():
+            ''' Begin reCAPTCHA validation '''
+            recaptcha_response = request.POST.get('g-recaptcha-response')
+            url = 'https://www.google.com/recaptcha/api/siteverify'
+            values = {
+                'secret': settings.GOOGLE_RECAPTCHA_SECRET_KEY,
+                'response': recaptcha_response
+            }
+            data = urllib.parse.urlencode(values).encode()
+            req = urllib.request.Request(url, data=data)
+            response = urllib.request.urlopen(req)
+            result = json.loads(response.read().decode())
+            ''' End reCAPTCHA validation '''
+            if result['success']:
+                user = form.save(commit=False)
+                user.username = user.username.lower()
```

Comment:
I end up with a 500 when attempting to make a user with a "duplicate" username as an existing user (case-sensitive bug).
<img width="1440" alt="Screen Shot 2021-02-26 at 2 15 01 PM" src="https://user-images.githubusercontent.com/30608004/109356025-5fc4c000-783d-11eb-8190-f7c6702b0a41.png">

Probably worth wrapping the `save` in a try-except and/or using the same pattern you use to get the Captcha and password errors through to the frontend.